packages/mediacenter/kodi-binary-addons/audiodecoder.usf/package.mk

```diff
@@ -17,4 +17,3 @@ PKG_LONGDESC="audiodecoder.usf"
 
 PKG_IS_ADDON="yes"
 PKG_ADDON_TYPE="kodi.audiodecoder"
-PKG_ADDON_PROJECTS="Generic Nvidia_Legacy RPi2 imx6 WeTek_Play"
```

Comment:
Looks like the above excluded RPi1 for some reason (maybe doesn't build for armv6, or not enough CPU) but now we will build this add-on for RPi1, or at least try...

Reply:
Fixed in #3531 - this add-on builds with SSE2 or NEON with no way to disable NEON so it fails to build on armv6.